workbench/app/Models/NullableTag.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Workbench\App\Models;
+
+use CalebDW\Laraflake\Concerns\HasSnowflakes;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\MorphTo;
+
+class NullableTag extends Model
```

Comment:
This model isn't actually used in a relationship and can probably be removed. Beyond testing that the macros produce the correct schema, this is starting to test the framework.